Lib/test/test_locale.py

```diff
@@ -576,6 +577,26 @@ def test_invalid_iterable_in_localetuple(self):
             locale.setlocale(locale.LC_ALL, (b'not', b'valid'))
 
 
+class TestGetFirstWeekDay(BaseLocalizedTest):
+
+    locale_type = locale.LC_ALL
```

Comment:
I haven’t checked how `locale_type` works here (it is not referenced in the test below), but shouldn’t the value depend on `LC_TIME`?  (see https://github.com/dmedvinsky/gsimplecal/issues/40 for example)